profiles/consts.py

```diff
@@ -1,4 +1,17 @@
 from django.utils.translation import ugettext_lazy as _
+from enumfields import Enum
+
+
+class NotificationType(Enum):
```

Comment:
well, consts are consts, enums are enums. `profiles/enums.py` is usually where those guys are.

Reply:
Indeed... splitting the file.